frontend/src/views/App/App.tsx

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
+import 'antd/dist/antd.css';
 import { StyledComponent } from './App.styled';
 
 export const App = () => {
   return (
     <div className="App">
-      <StyledComponent>
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload...
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </StyledComponent>
+      <StyledComponent />
```

Comment:
Jeśli to usunęliśmy, to żeby nam testy nie krzyczały na czerwono, to proponuję, aby tymczasowo pominąć test w pliku App.spec.tsx.

https://jestjs.io/docs/api#testskipname-fn

Reply:
done